src/app/components/modals/courseInfoModal/MoreEnrollInfo.tsx

```diff
@@ -0,0 +1,82 @@
+import { getQuarterColor } from "@/lib/quarterUtils";
+import { getMoreEnrollmentInfo } from "@actions/enrollment";
+import { StoredCourse } from "@customTypes/Course";
+import { Groups, LocationOn, People, Person } from "@mui/icons-material";
+import Info from "@mui/icons-material/Info";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionGroup,
+  AccordionSummary,
+  Card,
+  Chip,
+  Typography,
+} from "@mui/joy";
+import { useQuery } from "@tanstack/react-query";
+
+export interface MoreEnrollInfoProps {
+  course?: StoredCourse;
+}
+
+export default function MoreEnrollInfo({ course }: MoreEnrollInfoProps) {
+  const { data, isLoading } = useQuery({
+    queryKey: ["currentEnrollInfo", course?.departmentCode, course?.number],
+    queryFn: async () => await getMoreEnrollmentInfo(course!),
+    enabled: course !== undefined,
+  });
+
+  if (isLoading) return null;
+
+  if (data && data.length === 0) return null;
+
+  return (
+    <>
+      <Typography component="h2" level="h4" textColor="inherit" fontWeight="md">
```

Comment:
<img width="835" alt="image" src="https://github.com/SlugPath/SlugPath/assets/16377965/db7830b4-6681-48cf-bfd8-1282c201b9ca">
The title "Current & Upcoming Enrollment Information for" should have the same font weight and size as Quarters Offered, Credits etc titles for consistency.